python/services/evaluationservice/dmod/evaluationservice/test_config.json

```diff
@@ -0,0 +1,203 @@
+{
+    "observations": [
+        {
+            "name": "Observations",
+            "value_field": "observation",
+            "value_selectors": [
+                {
+                    "name": "observation",
+                    "where": "value",
+                    "path": ["values[*]", "value[*]", "value"],
+                    "datatype": "float",
+                    "origin": ["$", "value", "timeSeries[*]"],
+                    "associated_fields": [
+                        {
+                            "name":"value_date",
+                            "path": ["values[*]", "value[*]", "dateTime"],
+                            "datatype": "datetime"
+                        },
+                        {
+                            "name":"observation_location",
+                            "path": ["sourceInfo", "siteCode", "[0]", "value"],
+                            "datatype": "string"
+                        },
+                        {
+                            "name":"unit",
+                            "path": ["variable", "unit", "unitCode"],
+                            "datatype": "string"
+                        }
+                    ]
+                }
+            ],
+            "backend": {
+                "backend_type": "file",
+                "data_format": "json",
+                "address": "/Users/christopher.tubbs/PycharmProjects/DMOD/python/lib/evaluations/dmod/test/resources/observations.json"
```

Comment:
probably want to obfuscate this and all other `address` values.

Reply:
That file wasn't even supposed to be there in the first place. Good catch.